hostprocess/eventflow-logger/Dockerfile

```diff
@@ -1,5 +1,4 @@
-ARG OS=1809
-FROM mcr.microsoft.com/windows/nanoserver:${OS} as runtime
+FROM mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v0.1.0 as runtime
```

Comment:
this image is currently using Docker not buildx so I think it won't work. Will need to update the build scripts

Reply:
Yea, I noticed taht. Same w/ csi-proxy.
I might just update the images that are building w/ buildkit in the PR and then open a separate PR for these 2 images.